File: frontend/src/app/Models/workshop.ts
import { Colaborador } from "./Colaborador";


   export interface workshop
    {
    id: number;
    nome: string;
    dataRealizacao: string;
    descricao: string;
    presencas?: 
    {
    colaborador: Colaborador;
    registro: string;
    }[];
    }

File: backend/FastDesafio.Core/Workshop.cs
namespace FastDesafio.Core.Models
{
    public class Workshop{
        public int Id {get; set;}
        public string Nome {get; set;}
        public DateTime DataRealizacao {get; set;}
        public string Descricao {get; set;}
        public ICollection<Presenca> Presencas { get; set; } = new List<Presenca>();

    }
}